src/tests/example.spec.js

```diff
@@ -28,3 +28,83 @@ test.describe('Saucedemo app basic tests', () => {
         await expect(app.shoppingCart.cartItems).not.toBeAttached();
     });
 });
+
+test.describe('Saucedemo Unit 10 tests', () => {
+    test('sorting z-a', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
```

Comment:
You should definitely try typescript! But here you could try to move this type definition to the fixture so you dont need to duplicate this line for all occurrences.